opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -86,7 +84,7 @@ public synchronized void reset() {
     }
 
     /**
-     * @return a copy of all finish()ed MockSpans started by this MockTracer (since construction or the last call to
+     * @return all finish()ed MockSpans started by this MockTracer (since construction or the last call to
```

Comment:
Sorry, could we keep the original, mentioning this is a copy? ;) 

(Same for the new method)

Reply:
OK.
At the sametime, It is a copy of the `MockSpan` list not a deep copy of `MockSpans`.